### DISCUSSION TITLE
WallberryTheme and Pages Suggestion

### DISCUSSION THREAD

**Discussion by jdlucke** [delightedCrow/WallberryTheme#42] [Ideas]
Would it be possible to configure WallberryTheme to be different picture selections for different pages?  I am thinking that it would be interesting if I could place a sports-related image behind my sports modules pages, or a news theme behind the information pages.  You may already be able to do so, but I have not seen the config that would instigate it.  Any thoughts would be great.  Love the module.

**Comment by delightedCrow** [comment]
Thanks for the love :)

This sounds like a fun idea. Is there a specific module you're using for the pages functionality? I know it's a popular module and last I looked there were a few different forks/versions of it.

I vaguely recall someone doing something like this by adding multiple instances of the WallberryTheme in their config, then adding a different instance to each page, but I don't remember if it worked particularly well.

**Comment by jdlucke** [comment]
I use the MMM-pages and MMM-pages-indicator. Indicator seems to confuse
weather or clock modules for wallberry

**Comment by delightedCrow** [comment]
I'm guessing these two: [MMM-pages](https://github.com/edward-shen/MMM-pages) and [MMM-pages-indicator](https://github.com/edward-shen/MMM-page-indicator)?

Looks like MMM-pages does send socket notifications on page change. I had been considering adding functionality for doing background changes on notification events. I always figured I'd use it to change the background to match the current weather, but background on page change is also an interesting use case here 👍

**Comment by jdlucke** [comment]
Yes those are correct. I wish I had the coding skills to write modules.  35 years in IT but not coding 😄. So many cools things to do in todays tech. 



Jim Lucke
Director of Innovation
SmartReflectionsKC

***@***.***

9132109993

15116 West 132nd Street
Olathe, KS 66062

jimlucke.com











The content of this email is confidential and intended for the recipient specified in message only. It is strictly forbidden to share any part of this message with any third party, without a written consent of the sender. If you received this message by mistake, please reply to this message and follow with its deletion, so that we can ensure such a mistake does not occur in the future.

**Comment by delightedCrow** [comment]
The best time to start learning to code is when the right project makes it fun/worthwhile. Start with learning to make small changes and soon you'll be building whole modules in no time :D

I've added an issue for this feature [here that you can](https://github.com/delightedCrow/WallberryTheme/issues/43) follow if you like.

**Comment by senor-roberto** [comment]
For the OpenWeatherMap, could it be that a Personal API key is not good enough? Always get a 401 for it. Using the default Weather module in MMM with same API key does run normally.
